<launch>
    <!-- Static transform for ladybug panoramic camera -->
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="ladybug_static_transform" 
        args="-0.2 0 0 0 0 0 map ladybug_panoramic" /> -->

    <node name="ladybug_node" pkg="ladybug_driver" type="ladybug_node" output="screen">
        <!-- Existing parameters -->
        <param name="scale" value="100" />
        <param name="framerate" value="10" />
        <param name="jpeg_percent" value="80" />
        
        <!-- Trigger parameters -->
        <param name="trigger_enabled" value="false" />        <!-- Set to true to enable trigger mode -->
        <param name="trigger_delay" value="0.0" />          <!-- Trigger delay in milliseconds -->
        <param name="trigger_polarity" value="true" />      <!-- true for rising edge, false for falling edge -->
        <param name="trigger_timeout" value="5000" />       <!-- Timeout in milliseconds -->
        <param name="publish_individual_cameras" value="false" /> <!-- Set to true to publish individual cameras -->
    </node>
</launch>